change-validation/src/main/resources/template/ChangeValidationManagement.html

```diff
@@ -92,9 +92,28 @@
     </div>
 
     <div class="inner-portlet">
-      <div class="page-title">Configure Users with change validation</div>
+      <div class="page-title">Configure change request triggers</div>
       <div class="portlet-content">
-        <div class="col-xs-12 callout-fade callout-warning">
+        <div class="col-xs-12 callout-fade callout-info">
+          <div class="marker">
+            <span class="glyphicon glyphicon-info-sign"></span>
+          </div>
+          <p>
+            By default, change request are created for all users. You can change when a change request
+            is created with below options:
+          </p>
+          <ul>
+            <li>exempt some users from validation;</li>
+            <li>trigger change request only for changes impacting nodes belonging to some supervised groups;</li>
+          </ul>
+          <p>Be careful: a change request is created when <b>at least one</b> predicate matches, so an exempted user
+            still need a change request to modify a node from a supervised group.</p>
+        </div>
+      </div>
+
+      <div class="page-subtitle">Configure Users with change validation</div>
+      <div class="portlet-content">
+        <div class="col-xs-12 callout-fade callout-info">
           <div class="marker">
```

Comment:
Adding explanation about how predicate works because even with the doc, I spent 20 minutes not understanding why my changes weren't triggering when I thought they would.